sdk/eventgrid/arm-eventgrid/README.md

```diff
@@ -1,93 +1,101 @@
 ## Azure EventGridManagementClient SDK for JavaScript
 
-This package contains an isomorphic SDK for EventGridManagementClient.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for EventGridManagementClient.
```

Comment:
@deyaaeldeen Is the code gen still using lower case `n` here?

```suggestion
This package contains an isomorphic SDK (runs both in Node.js and in browsers) for EventGridManagementClient.
```

Reply:
Agh, my bad. I fixed the issue in the script but thought code gen did not have it. I opened https://github.com/Azure/autorest.typescript/pull/1075 with the fix.